README.md

```diff
@@ -73,6 +73,7 @@ USE_L10N = True
 # as found in the Transifex UI under your project
 TRANSIFEX_TOKEN = <project_token>  # used for pulling translations from Transifex
 TRANSIFEX_SECRET = <project_secret>  # used for pushing source content to Transifex
+TRANSIFEX_DAEMON_INTERVAL = <seconds> # used for defining the daemon running interval in seconds
```

Comment:
I'd prefer this to be something like `TRANSIFEX_SYNC_INTERVAL`, looks kinda clearer